packages/react-button/src/components/Button/useButtonVariants.ts

```diff
@@ -0,0 +1,175 @@
+import { ButtonVariants } from './Button.types';
+import { makeVariants, Theme } from '@fluentui/react-theme-provider';
+
+export const useButtonVariants = makeVariants<ButtonVariants>('Button', '--button', (theme: Theme) => {
+  const { fonts, effects, palette, semanticColors } = theme;
+
+  return {
+    base: {
+      size: {
+        smallest: '24px',
+        smaller: '24px',
+        small: '24px',
+        regular: '32px',
+        large: '40px',
+        larger: '48px',
+        largest: '64px',
+      },
+
+      // Sizing tokens
+      iconSize: fonts?.mediumPlus?.fontSize,
```

Comment:
```suggestion
      iconSize: fonts.mediumPlus.fontSize,
```

we should not need `?` everywhere

Reply:
pls sync code with latest code from `getTokens`